src/Service.Host/client/src/components/Menu.js

```diff
@@ -54,73 +53,45 @@ const theme = createMuiTheme({
     }
 });
 
-const menu = require('./menu.json').sections; // should get this via GET request to /intranet/menu
-
-const PinnedSubheaderList = ({ classes, list }) => (
-    <List className={classes.root} subheading={<li />}>
-        {list.map(item => (
-            <li key={item.title} className={classes.listSection}>
-                <ul className={classes.ul}>
-                    <ListSubheader>
-                        {item.title} <Divider />
-                    </ListSubheader>
-                    <div className={classes.innerList}>
-                        {item.items.map(entry => (
-                            <a href={entry.href} key={entry.title}>
-                                <ListItem>
-                                    <Typography color="primary"> {entry.title} </Typography>
-                                </ListItem>{' '}
-                            </a>
-                        ))}
-                    </div>
-                </ul>
-            </li>
-        ))}
-    </List>
+const MenuPage = ({ classes, section }) => (
+    <Page>
+        <MuiThemeProvider theme={theme}>
+            <List className={classes.root} subheading={<li />}>
+                {section !== null
+                    ? section.map(item => (
+                          <li key={item.title} className={classes.listSection}>
+                              <ul className={classes.ul}>
+                                  <ListSubheader>
+                                      {item.title} <Divider />
+                                  </ListSubheader>
+                                  <div className={classes.innerList}>
+                                      {item.items.map(entry => (
+                                          <a href={entry.href} key={entry.title}>
+                                              <ListItem>
+                                                  <Typography color="primary">
+                                                      {entry.title}
+                                                  </Typography>
+                                              </ListItem>
+                                          </a>
+                                      ))}
+                                  </div>
+                              </ul>
+                          </li>
+                      ))
+                    : ''}
```

Comment:
On first glance this looked a bit overwhelming with the nesting, but I think its fine if there's not a clearer way of doing it.